ecoscope_workflows/tasks/analysis/_aggregation.py

```diff
@@ -0,0 +1,25 @@
+from typing import Annotated
+
+import pandas as pd
+from pydantic import Field
+
+from ecoscope_workflows.annotations import DataFrame, JsonSerializableDataFrameModel
+from ecoscope_workflows.decorators import distributed
+
+
+@distributed
+def aggregate(
+    df: DataFrame[JsonSerializableDataFrameModel],
+    column_name: Annotated[str, Field(description="Start date")],
+    func_name: str,
```

Comment:
```suggestion
    func_name: Annotated[Literal["count", "mean", "sum"], Field(description="The method of aggregation".)
```
Perhaps we can constrain with a Literal here? So when called in `.replace(validate=True)` mode, Pydantic will throw a validation error if on of the specified literal values is not given.